novawallet/Common/Services/ExternalBalanceUpdater/CrowdloanService/CrowdloanOnChainSyncService.swift

```diff
@@ -119,6 +119,15 @@ final class CrowdloanOnChainSyncService: BaseSyncService {
     }
 
     override func performSyncUp() {
+        guard 
```

Comment:
I think we don't need the logic here. Because we need it on the upper level to guarantee that we even not create the service if the chain is disabled.